sitestats/pom.xml

```diff
@@ -19,8 +19,8 @@
   <properties>
     <commons-betwixt.version>0.8</commons-betwixt.version>
     <fop.version>1.0</fop.version>
-    <jfreechart.version>1.0.19</jfreechart.version>
-    <sst.wicket.version>6.30.0</sst.wicket.version>
+    <jfreechart.version>1.5.0</jfreechart.version>
+    <sst.wicket.version>6.26.0</sst.wicket.version>
```

Comment:
Can you remove the wicket version change?

Reply:
oops!